### PR TITLE
Disable sentry during local dev

### PR DESCRIPTION
**Pull request by arnifreyrs**
# \<Description\>

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/ui-shell/src/components/ErrorBoundary.tsx [hunk 1]
```diff
@@ -26,12 +26,14 @@ class ErrorBoundary extends PureComponent<Props, StateTypes> {
 
   componentDidCatch(error: Error) {
     const { application, currentScreen } = this.props
-    Sentry.withScope((scope) => {
-      scope.setExtra('applicationType', application.typeId)
-      scope.setExtra('applicationState', application.state)
-      scope.setExtra('currentScreen', currentScreen.id)
-      Sentry.captureException(error)
-    })
+    if (!window.location.origin.includes('http://localhost')) {
```
Comment (jeremybarbet) [inline libs/application/ui-shell/src/components/ErrorBoundary.tsx:29]: Why not using NODE_ENV variable?

Comment (arnifreyrs) [reply libs/application/ui-shell/src/components/ErrorBoundary.tsx:29]: because this is a react web app, not server side rendered

Comment (madebynoam) [reply libs/application/ui-shell/src/components/ErrorBoundary.tsx:29]: Could we do it like this?:
https://create-react-app.dev/docs/adding-custom-environment-variables/

Comment (madebynoam) [reply libs/application/ui-shell/src/components/ErrorBoundary.tsx:29]: > There is also a built-in environment variable called NODE_ENV. You can read it from process.env.NODE_ENV. When you run npm start, it is always equal to 'development', when you run npm test it is always equal to 'test', and when you run npm run build to make a production bundle, it is always equal to 'production'. You cannot override NODE_ENV manually. This prevents developers from accidentally deploying a slow development build to production.

Comment (arnifreyrs) [reply libs/application/ui-shell/src/components/ErrorBoundary.tsx:29]: I see the error in my ways, will fix, thx @jeremybarbet and @madebynoam

**Review by madebynoam — APPROVED** [review]
lgtm!

**Pull request merged**